src/election_authority_works/import/intermediate.clj

```
@@ -301,6 +301,59 @@
   (->> (maybe-string->keyword-or-vec val)
        (maybe-assoc-in row [:election-authority-level])))
 
+(defmethod parse-field :registration-form-url-en
+  [row _ val]
+  ;; we are only going to parse the -en field because there will never be a -es
+  ;; form url without an -en one, and taking them out of the row together is
+  ;; much easier than one at a time
+  (if (not (str/blank? val))
+    (let [rfuen val
+          rfes (:registration-form-url-es row)
+          _form {:en val}
+          form (maybe-assoc-in _form [:es] rfes)]
+      (assoc-in row [:voter-registration-methods :by-mail :registration-form-url] form))
+    row))
+
+(defmethod parse-field :registration-form-url-es
+  [row _ val]
+  ;; see comment on -en field
+  row)
+
+(defmethod parse-field :ballot-request-form-url-en
+  [row _ val]
+  ;; we are only going to parse the -en field because there will never be a -es
+  ;; form url without an -en one, and taking them out of the row together is
+  ;; much easier than one at a time
+  (if (not (str/blank? val))
+    (let [brfuen val
+          brfes (:ballot-request-form-url-es row)
+          _form {:en val}
+          form (maybe-assoc-in _form [:es] brfes)]
+      (assoc-in row [:voting-methods :by-mail :ballot-request-form-url] form))
+    row))
+
+(defmethod parse-field :ballot-request-form-url-es
+  [row _ val]
+  ;; see comment on -en field
+  row)
+
+(defmethod parse-field :party-instructions-en
+  [row _ val]
+  (maybe-assoc-in row [:instructions :party :en] val))
+
+(defmethod parse-field :party-instructions-es
+  [row _ val]
+  (maybe-assoc-in row [::instructions :party :es] val))
+
+(defmethod parse-field :early-voting-alias
+  [row _ val]
+  (if (not (str/blank?  val))
+    (let [ev-method (get-in row [:voting-methods :early-voting])]
+      (if ev-method
```

IMO we should throw an exception (which we'd get for free when we try to validate this row) if you try to import early-voting-alias in a state that doesn't support early voting (instead of ignoring that column).

---

I'd like to skip the validation for now.